streaming/src/lib/java/streaming_jni_common.h

```diff
@@ -0,0 +1,111 @@
+#ifndef RAY_STREAMING_JNI_COMMON_H
+#define RAY_STREAMING_JNI_COMMON_H
+
+#include <jni.h>
+#include <string>
+#include "ray/core_worker/common.h"
+#include "util/streaming_logging.h"
+
+class UniqueIdFromJByteArray {
```

Comment:
Shall we reuse jni common utility of ray? 
Or create a new common jni utility in ray.

Reply:
Yeah, I think we should do this. A lot of code is duplicated. cc @kfstorm